app/controllers/users.rb

```diff
@@ -0,0 +1,25 @@
+
+class Chitter < Sinatra::Base
+
+  get '/users/new' do
+    @user = User.new
+    erb :'users/new'
+  end
+
+  post '/users' do
+    @user = User.create(email: params[:email],
```

Comment:
Is there a way to avoid having to each piece of data (email, username etc) explicity?